<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{background:#ffffff;color:#1f1f1f;font:10pt &quot;Microsoft YaHei&quot;;}
QPlainTextEdit{background:#ffffff;color:#1f1f1f;border:1px solid #cfcfcf;border-radius:8px;padding:6px;}
QSpinBox{background:#ffffff;color:#1f1f1f;border:1px solid #cfcfcf;border-radius:6px;padding:4px 8px;}
QPushButton{background:#00a67d;color:#ffffff;border:none;border-radius:8px;padding:10px 14px;font-weight:600;}
QPushButton:hover{background:#009a74;}
QPushButton:pressed{background:#008a67;}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutRoot">
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>服务器端口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxPort">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>45454</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelOnline">
        <property name="text">
         <string>在线人数：0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEditLog">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout">
      <item>
       <spacer name="horizontalSpacerBottom">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStartStop">
        <property name="text">
         <string>启动服务器</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
